docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -1502,6 +1502,18 @@ CREATE TABLE example.customers.orders (
 WITH (sorted_by = ARRAY['order_date'])
 ```
 
+You can explicitly configure sort directions or null ordering in the following way.
+
+```
+CREATE TABLE example.customers.orders (
+    order_id BIGINT,
+    order_date DATE,
+    account_number BIGINT,
+    customer VARCHAR,
+    country VARCHAR)
+WITH (sorted_by = ARRAY['order_date DESC NULLS FIRST', 'order_id ASC NULLS LAST'])
+```
+
```

Comment:
> Explain how to specify sort directions and null ordering using the sorted_by property

The commit message is too long. Could you change to "Document sort direction and null order in Iceberg" or something? 

https://trino.io/development/process#pull-request-and-commit-guidelines-
> Limit the subject line to 50 characters.